Insert Nomadic Labs in the list of companies using OCaml (#1187)
ocaml/ocaml.org: site/learn/companies.md

    </dd>
</dl>

<dl class="row">
    <dt class="span3">
        <img src="/img/users/nomadic-labs.png" alt="Nomadic Labs">
    </dt>
    <dd class="span5">
        <h2><a href="https://www.nomadic-labs.com">Paris</a>, France</h2>
        <p>

>         Nomadic Labs houses a team focused on Research and Development. Our core competencies are in programming language theory and practice, distributed systems, and formal verification. We believe our strength lies in a unique mix of skills and experience, allowing us to transfer the best of academic research into real world applications.

Reviewer: Maybe we could give a hint about the size of the company by providing the number of researchers.
```suggestion
        Nomadic Labs brings together more than 40 engineering researchers whose core competencies are in programming language theory and practice, distributed systems, and formal verification. We believe our strength lies in a unique mix of skills and experience, allowing us to transfer the best of academic research into real world applications.
```

> Nomadic Labs focuses on contributing to the development of the Tezos core software, including the smart-contract language, Michelson. We also have a mobile team working on the Cortez wallet (Android, iOS).

Reviewer: I would suggest to :
1. add links to the tezos OCaml implementation and Nomadic michelson documentation
2. remove the Cortez part since this one is not written in OCaml and may not be relevant

```suggestion
Nomadic Labs focuses on contributing to the development of the <a href="https://gitlab.com/tezos/tezos/">Tezos</a> core software, including the smart-contract language, <a href="https://michelson.nomadic-labs.com/">Michelson</a>.
```

    </dd>
</dl>

<dl class="row">
    <dt class="span3">
        <img src="/img/users/nomadic-labs.png" alt="Nomadic Labs">
    </dt>
    <dd class="span5">

>         <h2><a href="https://www.nomadic-labs.com">Paris</a>, France</h2>

Reviewer: ```suggestion
        <h2><a href="https://www.nomadic-labs.com">Nomadic Labs</a>, France</h2>
```

        <p>

Nomadic Labs houses a team focused on Research and Development. Our
core competencies are in programming language theory and practice,
distributed systems, and formal verification. Nomadic Labs focuses on
contributing to the development of the Tezos core software, including
the smart-contract language, Michelson.

Tezos infrastructure is entirely implemented in OCaml. It strongly
relies on OCaml efficiency and expressivity. For instance, Michelson
smart contracts are represented using OCaml GADTs to prevent many

> runtime errors to happen. Safety and correctness are critical for a

Reviewer 2: to prevent many runtime errors to happen -> to prevent many runtime errors from happening

Author: Thank you @pw374 !
